orange/src/main/java/com/hartwig/hmftools/orange/report/components/Footer.java

```diff
@@ -10,56 +10,66 @@
 import com.itextpdf.kernel.pdf.canvas.PdfCanvas;
 import com.itextpdf.kernel.pdf.xobject.PdfFormXObject;
 import com.itextpdf.layout.Canvas;
-import com.itextpdf.layout.Style;
 import com.itextpdf.layout.element.Paragraph;
 import com.itextpdf.layout.property.TextAlignment;
 
 import org.jetbrains.annotations.NotNull;
 
 public class Footer {
 
-    private final List<PageNumberTemplate> pageNumberTemplates = Lists.newArrayList();
+    private final List<FooterTemplate> footerTemplates = Lists.newArrayList();
     private final ReportResources reportResources;
+    private final boolean addDisclaimer;
 
-    public Footer(ReportResources reportResources) {
+    public Footer(ReportResources reportResources, boolean addDisclaimer) {
         this.reportResources = reportResources;
+        this.addDisclaimer = addDisclaimer;
     }
 
     public void renderFooter(@NotNull PdfPage page) {
         PdfCanvas canvas = new PdfCanvas(page.getLastContentStream(), page.getResources(), page.getDocument());
 
         int pageNumber = page.getDocument().getPageNumber(page);
-        PdfFormXObject pageNumberTemplate = new PdfFormXObject(new Rectangle(0, 0, 200, 20));
+        PdfFormXObject pageNumberTemplate = new PdfFormXObject(new Rectangle(0, 0, 450, 20));
         canvas.addXObject(pageNumberTemplate, 58, 20);
-        pageNumberTemplates.add(new PageNumberTemplate(pageNumber, pageNumberTemplate));
+        footerTemplates.add(new FooterTemplate(pageNumber, pageNumberTemplate, addDisclaimer));
 
         canvas.release();
     }
 
-    public void writeTotalPageCount(@NotNull PdfDocument document) {
+    public void writePageCounts(@NotNull PdfDocument document) {
```

Comment:
Similar comment as for `PageEventHandler`: Shouldn't this function be called `writeFooters`?